hooks/useUser.js

```diff
@@ -13,23 +13,25 @@ export default function useUser() {
     fetcher
   );
 
-  async function login({ email, password }) {
+  async function login({ email, roomNumber }) {
     const response = await fetch("/api/users/login", {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
-      body: JSON.stringify({ email, password }),
+      body: JSON.stringify({ email, roomNumber }),
     });
 
     if (!response.ok) {
-      alert("Wrong credentials!");
+      alert("Incorrect Credentials");
       return;
     }
 
     const json = await response.json();
```

Comment:
I suggest a more descriptive name than `json` as this could be anything. You are saving the user data of the logged-in user in this `const`, so I suggest a name like `loggedInUser` or something similiar.